components/tests/python/library/__init__.py

```diff
@@ -214,6 +214,12 @@ def new_image(self, name=""):
         img.acquisitionDate = rtime(0)
         return img
 
+    def new_dataset(self, name="", description=None):
```

Comment:
What's the reasoning for adding this helper method as opposed to using `ITest.make_dataset()` ?